hack/tools/swagger/lib/render.go

```diff
@@ -80,6 +83,8 @@ func RenderOpenAPISpec(cfg Config) (string, error) {
 	}
 
 	serverConfig := genericapiserver.NewRecommendedConfig(cfg.Codecs)
+	serverConfig.ClientConfig = &clientgorest.Config{}
+	_ = utilfeature.DefaultMutableFeatureGate.Set(fmt.Sprintf("%s=%v", features.APIPriorityAndFairness, false))
```

Comment:
@chaosi-zju 
Can you help to confirm if it is OK? and help to confirm the generated `swagger.json`?